### PR TITLE
Some minor corrections

### PR DESCRIPTION
**Pull request by atishamte**
1. Spelling correction
2. Function return type was missing
3. Code optimization

Each pull request should address a single issue, and have a meaningful title.

**Description**
Explain what you have changed, and why.

**Checklist:**
- [x] Securely signed commits
- [x] Component(s) with PHPdocs
- [x] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

---------Remove from here down in your description----------

**Notes**
- Pull requests must be in English
- If the PR solves an issue, reference it with a suitable verb and the issue number
(e.g. fixes <hash>12345
- Unsolicited PRs will be considered, but there is no guarantee of acceptance
- Pull requests should be from a feature branch in the contributor's fork of the repository
  to the develop branch of the project repository

### CODE REVIEW

File: system/Helpers/text_helper.php [hunk 1]
```diff
@@ -492,7 +492,7 @@ function word_wrap(string $str, int $charlim = 76): string
 			}
 		}
 
-		// Use PHP's native function to do the initial wordwrap.
+		// Use PHPs native function to do the initial wordwrap.
```
Comment (jim-parry) [inline system/Helpers/text_helper.php]: The original is correct. "PHP's" is possessive.

File: system/Helpers/text_helper.php [hunk 2]
```diff
@@ -756,9 +756,8 @@ function random_string(string $type = 'alnum', int $len = 8): string
 
 				return substr(str_shuffle(str_repeat($pool, ceil($len / strlen($pool)))), 0, $len);
 			case 'md5':
-				return md5(uniqid(mt_rand(), true));
 			case 'sha1':
-				return sha1(uniqid(mt_rand(), true));
+				return $type(uniqid(mt_rand(), true));
```
Comment (jim-parry) [inline system/Helpers/text_helper.php]: This may be more "efficient", but the original is clearer

**Comment by jim-parry** [comment]
1) You broke a couple of unit tests with your changes
2) I have added a couple of comments to the changes
3) contributions should be GPG-signed, for proper attribution

File: system/Helpers/text_helper.php (hunk 1 above)
Comment (atishamte) [reply system/Helpers/text_helper.php]: Made changes

File: system/Helpers/text_helper.php (hunk 2 above)
Comment (atishamte) [reply system/Helpers/text_helper.php]: Reverted the changes

**Pull request closed without merging**